data/mil2525d/utilities/military-features-utilities/README.md

```diff
@@ -20,23 +23,25 @@
 
 ## Instructions
 
-### Create Military Features Geodatabase Template
+### Create Military Feature Geodatabase Template
 
 #### Overview
 
 The utility creates a new File Geodatabase (GDB) in a specified folder, replacing any existing geodatabase of the same name in that folder.  The version of the newly created geodatabase can also be specified.
 
-The structure of the geodatabase is determined by the contents of the CSV files found in a  folder selected at run-time.  The most recent version of these CSV files can be found [here](https://github.com/Esri/joint-military-symbology-xml/tree/master/samples/military_feature_schemas).
+The structure of the geodatabase is determined by the contents of the CSV files found in a schema folder selected at run-time.  The most recent version of these CSV files can be found [here](https://github.com/Esri/joint-military-symbology-xml/tree/master/samples/military_feature_schemas).
```

Comment:
From here down looks like a really good, thorough tech review+update of this doc/readme - nice job